opentracing-shim/src/main/java/io/opentelemetry/opentracingshim/ScopeManagerShim.java

```diff
@@ -18,22 +22,31 @@ public ScopeManagerShim(TelemetryInfo telemetryInfo) {
   @Override
   @SuppressWarnings("ReturnMissingNullable")
   public Span activeSpan() {
+    Context context = Context.current();
+    io.opentelemetry.api.trace.Span span = io.opentelemetry.api.trace.Span.fromContext(context);
+    SpanShim spanShim = context.get(SPAN_SHIM_KEY);
```

Comment:
Any thoughts to making `SpanShim` implement `ImplicitContextKeyed`, or do you not want to expose that API on the shim?

Reply:
Yes, definitely. Left it out to make it easier for Yuri (or whoever reviews that Spec PR) to review.

Reply:
Do you think we can have `SpanShim` just implement both OTel and OT `span` and not have to keep track of two objects? It worked ok in OC so I suspect it'll work for OT.

https://github.com/open-telemetry/opentelemetry-java/blob/main/opencensus-shim/src/main/java/io/opentelemetry/opencensusshim/OpenTelemetrySpanImpl.java#L53

Reply:
I can take a stab at such optimization once the OT Spec portion is merged ;)

Reply:
From the perspective of maintainence here, if both shims operated very similarly that would be quite great. Is it possible to block work on the OT shim until the spec is merged? Or alternatively, do you have to wait for the spec? I think the only way to spec out the shims is to implement them first, as @jsuereth did with OC I believe.